src/Search/Search.Management.Tests/packages.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
(The git web page can't show all of the changes here, so my suggestion might not help)
if you depend on the TestDependencies like most other ones do, then you don't need to have this file.